Create_Volume_From_Snapshot.md

```diff
@@ -0,0 +1,84 @@
+Create Volumes from Snapshots
+
+**Status**: Pending
+
+**Version**: Alpha
+
+**Author(s)**: Xiaoyan Li (@lixiaoy1)
+
+## Summary
+
+This proposal is drafted for creating a volume from snapshot supported backend storages. It defines the APIinterfaces to OpenSDS customers and meanwhile the interfaces for kinds of storage drivers.
```

Comment:
Missing space between "API" and "interfaces".

Reply:
done.